src/Components/Projects/Projects.tsx

```diff
@@ -153,179 +141,184 @@ const Projects: React.FC = () => {
 
   useEffect(() => {
     getAllProjectsList();
-  }, [])
+  }, []);
   return (
     <>
-    <div className="header d-flex justify-content-between p-3">
-      <h3>projects</h3>
-      <button
-        onClick={navigateToNew}
-        className="btn btn-warning rounded-5 px-4"
-      >
-        <i className="fa fa-plus" aria-hidden="true"></i> &nbsp;Add New
-        Project
-      </button>
-    </div>
-    {/* table */}
-    <>
-      <div className="table-container1 vh-100">
-        <table className="table">
-          <thead className="table-head table-bg ">
-            <tr>
-              <th scope="col">Title</th>
-              <th scope="col">Description</th>
-              <th scope="col">Num Task</th>
-              <th scope="col">Actions</th>
-            </tr>
-          </thead>
-          <tbody>
-            {projects?.length > 0 ? (
-              projects.map((project: any) => (
-                <tr key={project?.id}>
-                  <td>{project?.title}</td>
-                  <td>{project?.description}</td>
-                  <td>{project?.task?.length}</td>
-                  <td>
-                    <i
-                      onClick={() => showViewModal(project?.id)}
-                      className="fa fa-eye  text-info px-2"
-                    ></i>
-                    <i
-                      onClick={() => showUpdateModal(project)}
-                      className="fa fa-pen  text-warning px-2"
-                    ></i>
-                    <i
-                      onClick={() => showDeleteModal(project.id)}
-                      className="fa fa-trash  text-danger"
-                    ></i>
+      <div className="header d-flex justify-content-between p-3">
+        <h3>projects</h3>
+
+        <button className="btn btn-warning rounded-5 px-4 ">
```

Comment:
You can remove the button element and add its styles to Link element, since both of them are interactive elements, and no need to nest them inside each other. and nesting them might have  potential issues since it is a violation of the html specification:

![image](https://github.com/Rawan-Mohamed/project-management-system-1/assets/64327685/75798199-9641-4a0e-905b-ddc2b9c19b0c)

Reply:
done